rel_scripts/gcb/gcb_lib.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+# This script is meant to be sourced, has a set of functions used by scripts on gcb
+
+#sets GITHUB_KEYFILE to github auth file
```

Comment:
all this code needs to be merged into istio/istio this is a temporary copy so that we can run builds on master
once everything is done it will go away.

Reply:
why do we need to merge this in this repo temporarily? why not merge to istio/istio/release/new directly?

Also, I think the new pipeline should be based on a new repo, not "daily-release".

Reply:
I would much rather to create a new repo now for the new pipeline and leave the existing one for the old pipeline in the meantime, instead of mixing both tWhaogether.

What we need from this repo are only the files in prow/, and add a new prow config, right?


Also, this repo should only have the trigger files, and I suppose that should be the design principle. Adding all these scripts here violates that.